Source/Fuse.Common/Tests/FuseTest/ObservableCollector.uno

```diff
@@ -55,7 +55,7 @@ namespace FuseTest
 			CleanSubscription();
 			if (_items == null)
 				return;
-			_subscription = _items.Subscribe(this);
+			_subscription = (ISubscription)_items.Subscribe(this);
```

Comment:
@Duckers: So, are you saying that in this case, it's guaranteed? I mean, the test will obviously fail if it doesn't, it just sounds a bit fragile...